### DISCUSSION TITLE
Identifying the corresponding elements in Revit and IFC from the RDF data

### DISCUSSION THREAD

**Discussion by ajayka7** [jyrkioraskari/IFCtoLBD#84] [Q&A]
Hi, 

I have created a simple Revit model, converted it into IFC, and then converted it into RDF using the IFCtoLBD converter. In the resulting RDF file, I can not understand which data corresponds to which element in Revit and IFC because the elements have some unique way of representation, for example - inst:wall_1eec9390-5f68-4a64-aaa9-34a883f3c3bf_geometry_bb. I want to query the RDF to extract some data from it, however, as I can not identify the elements correctly, I cannot draft the queries properly. 

Is there any way to identify the corresponding elements in Revit and IFC based on their representation in the generated RDF data?

Thanks,
Ajay

**Comment by jyrkioraskari** [comment]
Hi Ajay,
One of the methods is to use the IFC GUID parameter, which is a unique identifier for each element in the IFC file. The IFC GUID parameter can be added to Revit elements as a shared parameter, and then exported to the IFC file as a user-defined property. 
The parameter can be found in an exported LBD file pointed by  the props:globalIdIfcRoot_attribute_simple  property when using PROPS Level 1.

Kind regards,
J.
